charts/celestia-local/files/scripts/init-bridge.sh

```diff
@@ -14,3 +14,6 @@ celestia bridge init \
   --core.grpc.port $celestia_app_grpc_port \
   --gateway.port $bridge_host_port
 cp -r "$home_dir/keyring-test" "$home_dir/bridge/keys/"
+
+sed -i 's/PeersLimit = 5/PeersLimit = 1/' $home_dir/bridge/config.toml
+sed -i 's/Low = 50/Low = 0/' $home_dir/bridge/config.toml
```

Comment:
We don't need celestia local to have any peers, it shouldnt have any it's a one off instance.